test/blackbox-tests/test-cases/actions/concurrent-multi-diff.t/run.t

```diff
@@ -0,0 +1,84 @@
+The use of (concurrent ) illustrated in the context of diffing multiple files.
+
+Say we want to diff 3 files.
+  $ cat A
+  I am file A.
+  $ cat B
+  I am file B.
+  $ cat C
+  I am file C.
```

Comment:
Should these (and those code examples following) be in a code block?

Reply:
No because this is a cram test and does not have any formatting like that.